### PR TITLE
Add the Performance Analysis Results summary table

### PR DESCRIPTION
**Pull request by RukshiW**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: en/streaming-integrator/docs/ref/performance-analysis-results.md [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
+
+- **Xms**: 2GB
+
+- **Xmx**: 4GB
+
+The exact specifications used in the use cases listed in this section are summarised in the table below:
+
+| **Scenario**                                          | **CPU**   | **Memory** | **Input TPS** | **Input Message Size** | **Total Message Count** | **Output TPS** |
+|-------------------------------------------------------|-----------|------------|---------------|------------------------|-------------------------|----------------|
+| [Consuming events using Kafka source](#consuming-events-using-kafka-source)| | | 180K | | | 1.1M |
```
Comment (ramindu90) [inline en/streaming-integrator/docs/ref/performance-analysis-results.md:23]: Use the https://aws.amazon.com/ec2/instance-types/#Compute_Optimized 's c5 column to check the CPU and Memory details that was used

**Review by ramindu90 — CHANGES_REQUESTED** [review]

File: en/streaming-integrator/docs/ref/performance-analysis-results.md [hunk 2]
```diff
@@ -1,7 +1,34 @@
...
+
+The recommended CPU and memory specifications for Docker containers are as follows:
+
+- **CPU**: 4 cores
+
+- **Memory**: 8GB
+
+- **Xms**: 2GB
+
+- **Xmx**: 4GB
```
[the remark this answers is not in the document]
Comment (RukshiW) [reply en/streaming-integrator/docs/ref/performance-analysis-results.md:17]: Added a separate column for this.

File: en/streaming-integrator/docs/ref/performance-analysis-results.md (hunk 1 above)
Comment (RukshiW) [reply en/streaming-integrator/docs/ref/performance-analysis-results.md]: Added the CPU and memory details from the link provided. Thank you.

File: en/streaming-integrator/docs/ref/performance-analysis-results.md [hunk 3]
```diff
@@ -3,6 +3,33 @@
...
```
Comment (ramindu90) [inline en/streaming-integrator/docs/ref/performance-analysis-results.md:17]: This content has to come under 
"The recommended memory specifications SI server as follows and should be configured in {SI_Home}/wso2/server/bin/carbon.sh

**Review by ramindu90 — CHANGES_REQUESTED** [review]

**Review by ramindu90 — APPROVED** [review]

File: en/streaming-integrator/docs/ref/performance-analysis-results.md (hunk 3 above)
Comment (RukshiW) [reply en/streaming-integrator/docs/ref/performance-analysis-results.md:17]: Added the mentioned text.

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**